packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -907,9 +913,72 @@ class SelectionOverlay {
   /// Disposes this object and release resources.
   /// {@endtemplate}
   void dispose() {
+    _androidPostDragShowTimer?.cancel();
+    _androidPostDragShowTimer = null;
     hide();
   }
 
+  bool _startHandleDragging = false;
+  bool _endHandleDragging = false;
+  bool get _draggingHandles => _startHandleDragging || _endHandleDragging;
+
+  Future<void> _handleDragging(bool startedDragging) async {
+    // Hide toolbar while dragging either handle on Android and iOS.
+    const Set<TargetPlatform> mobilePlatforms = <TargetPlatform>{ TargetPlatform.iOS, TargetPlatform.android };
```

Comment:
Nit: [Elsewhere](https://master-api.flutter.dev/flutter/flutter_test/TargetPlatformVariant/TargetPlatformVariant.mobile.html) we include fuchsia in mobile platforms.  Maybe use a different name or just use the Set inline?  Not a big deal either way.

Reply:
I moved it to a final variable to check for it in `showToolbar` as well.